proj/src/main/webapp/script.js

```diff
@@ -429,6 +437,36 @@ function addUsernameToMessage(uid, elementId) {
     })
 }
 
+function initBio() {
+    const bioBox = document.getElementById('user-bio');
+    const editInputBox = document.getElementById('bio-edit');
+
+    bioBox.addEventListener('dblclick', function() {
+        this.hidden = true;
+        const bio = this.innerText;
+
+        editInputBox.hidden = false;
+        editInputBox.innerText = bio;
```

Comment:
You could remove `bio` as the middle man and just do `editInputBox.innerText = this.innerText;` as you did on the line below this.

Reply:
actually i just realized that that line of code doesn't do anything so i just removed it